package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "five-bells-notary",
-  "version": "2.0.2",
```

Comment:
My understanding is that we bump a major version if the API we are providing changes, to indicate to the other components that this change will break their code.  

This one is changing the API Notary uses (rather than it provides), so a minor change would do, I believe.